src/app/components/sidebar/create-community-modal/create-community-modal.component.scss

```diff
@@ -0,0 +1,42 @@
+:host {
+  display: none;
+}
+
+.create-community-modal {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+
+  h2 {
+    margin-right: auto;
+    margin-bottom: 0;
```

Comment:
Maybe it will be better to make refresh all styles like `* { margin : 0, padding: 0 .... ` in main styles file for all tags